api/apps/api/config/custom-environment-variables.json

```diff
@@ -4,12 +4,22 @@
   },
   "postgresApi": {
     "url": "API_POSTGRES_URL",
+    "host": "API_POSTGRES_HOST",
+    "username": "API_POSTGRES_USER",
+    "password": "API_POSTGRES_PASSWORD",
+    "database": "API_POSTGRES_DB",
+    "port": "API_POSTGRES_PORT",
```

Comment:
`API_POSTGRES_URL` already includes host, user, password, db and port - can we rather compose this single envvar when running natively so that we avoid having duplication of pg connection configs?

Reply:
upon closer inspection on reverting my changes, I actually noticed that API_POSTGRES_URL is a sort of internal variable, as devs are always expected to provide the individual host/user/password/db values. So, in DRY spirit, I'm removing the URL var instead